src/main/java/com/github/fishio/Entity.java

```diff
@@ -80,12 +80,12 @@ public void kill() {
 	
 	
 	@Override
-	public ICollisionArea getBoundingArea() {
-		return boundingArea;
+	public CollisionMask getBoundingArea() {
+		return (CollisionMask) boundingArea;
```

Comment:
Unnecessary cast. Just change the type of the boundingArea attribute from ICollisionArea to CollisionMask.

<logic